### PR TITLE
feat(onboarding): Import/init Sentry on top

### PR DESCRIPTION
**Pull request by s1gr1d**
Updates the serverless snippets to make the top-import of Sentry more clear.

### CODE REVIEW

**Review by Lms24 — APPROVED** [review]

File: static/app/gettingStartedDocs/node/gcpfunctions.tsx [hunk 1]
```diff
@@ -17,7 +17,9 @@ import {getInstallConfig, getSdkInitSnippet} from 'sentry/utils/gettingStartedDo
 type Params = DocsParams;
 
 const getSdkSetupSnippet = (params: Params) => `
+// IMPORTANT: Make sure to import Sentry at the top of your file.
```
Comment (mydea) [inline static/app/gettingStartedDocs/node/gcpfunctions.tsx:20]:
```suggestion
// Make sure to initialize Sentry
```

Not sure about the "IMPORTANT" part, feels a bit "in your face" but maybe this is fine :D But we should reword this to say initialize instead of import!

File: static/app/gettingStartedDocs/node/azurefunctions.tsx [hunk 2]
```diff
@@ -18,7 +18,9 @@ type Params = DocsParams;
 
 const getSdkSetupSnippet = (params: Params) => `
 "use strict";
+// IMPORTANT: Make sure to import Sentry at the top of your file.
```
Comment (mydea) [inline static/app/gettingStartedDocs/node/azurefunctions.tsx:21]:
```suggestion
// Make sure to initialize Sentry at the top of your file
```

see comment below

File: static/app/gettingStartedDocs/node/awslambda.tsx [hunk 3]
```diff
@@ -17,7 +17,9 @@ import {getInstallConfig, getSdkInitSnippet} from 'sentry/utils/gettingStartedDo
 type Params = DocsParams;
 
 const getSdkSetupSnippet = (params: Params) => `
+// IMPORTANT: Make sure to import Sentry at the top of your file.
```
Comment (mydea) [inline static/app/gettingStartedDocs/node/awslambda.tsx:20]:
```suggestion
// Make sure to initialize Sentry at the top of your file
```

see comment below

**Review by mydea — APPROVED** [review]
